alibi_detect/od/pytorch/ensemble.py

```diff
@@ -18,93 +17,63 @@ def __init__(self):
         super().__init__()
 
     def transform(self, x: torch.Tensor):
```

Comment:
Just wondering if there's a reason not to make this an `abstracmethod` and inherit from `ABC`?

Reply:
Ah I wasn't aware that you have to initialize it in the child class to appease `torch`, that makes sense then. Definitely not initializing abstract classes (wouldn't work anyway!).